webpack/components/forms/FormShow.js

```diff
@@ -126,9 +126,7 @@ class FormShow extends Component {
             <Link className="btn btn-default" to={`/forms/${form.id}/extend`}>Extend</Link>
           }
           <button className="btn btn-default" onClick={() => window.print()}>Print</button>
-          {this.props.currentUser && this.props.currentUser.id &&
-            <a className="btn btn-default" href={Routes.redcapFormPath(form)}>Export to Redcap</a>
-          }
+          <a className="btn btn-default" href={`/forms/${form.id}/redcap`}>Export to Redcap</a>
```

Comment:
jenkins failing due to linter issue from this change
```

/home/jenkins/workspace/P-Vocabulary-Service_PR-380-2FPOI7YSRCHRMCRWC2DMQACOG56AAJE73DWEXGY6KITSMKK42JFA/webpack/components/forms/FormShow.js
  8:8  error  'Routes' is defined but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)
```

Reply:
Forgot to include , pushed and waiting for a rebuild